beta-src/src/components/map/components/WDTerritory.tsx

```diff
@@ -43,6 +43,8 @@ const WDTerritory: React.FC<WDTerritoryProps> = function ({
     (state) => state.game.commands.territoryCommands[territoryMapData.name],
   );
 
+  const isFleetClicked = useAppSelector((state) => state.game.isFleetClicked);
```

Comment:
Can we use commands for this? This is going to cause every territory to re-render whenever this flag changes.  Ideally when we know we need to activate the coastal pieces we should check which territories need to re-render and only target those.

Reply:
@bryancodazen - Can we sync up on direction for this?